server/models/spaceModel.ts

```diff
@@ -0,0 +1,9 @@
+import { Typings } from "../typings";
+
+export const spaceModel = (data: Typings.RemoteSpaceData) => {
```

Comment:
This is a weird name for this, I would call it `buildIssPositionData` or something like that. Also, you're missing a return signature (you'll need to make a new interface, because your service should also be returning the same shape of data)